### ISSUE TITLE
Review the "dubious import / exports" and assess what should actually be imported/exported

### ISSUE THREAD

**Issue by wdanilo** [enso-org/enso#4755]
This task is automatically imported from the old Task Issue Board and it was originally created by James Dunkerley.
Original issue is [here](https://www.pivotaltracker.com/story/show/183452411).
<hr />

Make a design of what changes we want and then agree tickets to change them.

## Tasks:
- [X] Base
- [X] Database
- [X] Examples
- [X] Geo
- [X] Google_Api
- [X] Image
- [X] Table
- [X] Test
- [X] Visualisation
- [X] test_scripts

## Comments:
**James Dunkerley** reports a new **STANDUP** for today (2022-10-06):

**Progress:** Worked through tidying all of Database... It should be finished by 2022-10-07.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). Prototype work on expressions (Enso Bot - Oct 6, 2022)
<hr />
**James Dunkerley** reports a new **STANDUP** for today (2022-10-07):

**Progress:** Finished Database restructure. Worked on expression prototype idea. It should be finished by 2022-10-07.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). Write up expressions prototype. (Enso Bot - Oct 7, 2022)
<hr />
**James Dunkerley** reports a new **🔴 DELAY** for today (2022-11-16):

**Summary:** There is **45 days** delay in implementation of the [Review the "dubious import / exports" and assess what should actually be imported/exported (#183452411) task](https://www.pivotaltracker.com/story/show/183452411).
It **will cause 0 days delay** for the delivery of this weekly plan.

Picking up completing this work on Base.

**Delay Cause:** Been on hold for a while - but need it done so can move forward. (Enso Bot - Nov 16, 2022)
<hr />
**James Dunkerley** reports a new **STANDUP** for today (2022-11-16):

**Progress:** Finished the work on the Nothing within a Vector.distinct. Passing all the tests now. Handed over to Hubert to look at engine changes and sign off. Figma design work with Radek on Join and review with Ned and Sylwia. Interview with Sylwia. Started moving everything around in Base. It should be finished by 2022-11-21.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). Working from London for variety and continuing with the Base statics and types. (Enso Bot - Nov 16, 2022)
<hr />
**James Dunkerley** reports a new **STANDUP** for today (2022-11-17):

**Progress:** Big batch and first PR prepared for the Standard.Base tidy. Caught up with old colleague working in private equity and discussing whether a good fit for enso within their analytics department, going to reach out next week and see if we can talk with their head of analytics. Caught up with TRG headhunter. It should be finished by 2022-11-21.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). More tidying... (Enso Bot - Nov 17, 2022)
<hr />
**James Dunkerley** reports a new **STANDUP** for today (2022-11-18):

**Progress:** Second batch of updates to Standard.Base done. Design session with Ned on Nothing and Null. It should be finished by 2022-11-21.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). More tidying... (Enso Bot - Nov 18, 2022)
<hr />
**James Dunkerley** reports a new **STANDUP** for today (2022-11-19):

**Progress:** a few hours (4) on the third batch of updates. It should be finished by 2022-11-21.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). More tidying... (Enso Bot - Nov 19, 2022)
<hr />
**James Dunkerley** reports a new **STANDUP** for today (2022-11-20):

**Progress:** a few hours (4) on the third batch of updates. Most of the Time types now done (just 2 more to go - will do tomorrow morning). Aiming for part 3 to be ready about stand up time. It should be finished by 2022-11-21.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). Moving onto Range and Text (the last big ones left) at which point Standard.Base.Data should be done. Hopefully other areas of Base will be less painful. (Enso Bot - Nov 20, 2022)
<hr />
**James Dunkerley** reports a new **STANDUP** for today (2022-11-21):

**Progress:** Completed the 3rd PR. Found problems with the Parse_Values_Spec which was not being included. This meant various issues in the parse_values function. Resolved those issues. It should be finished by 2022-11-21.

**Next Day:** Next day I will be working on the [#183452411 task](https://www.pivotaltracker.com/story/show/183452411). Getting through the 4th PR once Dmitry's PR is merged. (Enso Bot - Nov 21, 2022)
<hr />
